xbmc/addons/AddonManager.cpp

```diff
@@ -320,6 +320,24 @@ bool CAddonMgr::Init()
 
   FindAddons();
 
+  // disable some system addons by default because they are optional
+  VECADDONS addons;
+  GetAddons(ADDON_PVRDLL, addons);
+  GetAddons(ADDON_AUDIODECODER, addons);
+  GetAddons(ADDON_AUDIOENCODER, addons);
+  std::string systemAddonsPath = CSpecialProtocol::TranslatePath("special://xbmc/addons");
+  for (size_t i = 0; i<addons.size(); i++)
+  {
```

Comment:
But you introduce new code and we should use modern coding style and this is a no-brainer imo